Formula/pokerstove.rb

```diff
@@ -0,0 +1,38 @@
+class Pokerstove < Formula
+  desc "Poker evaluation and enumeration software"
+  homepage "https://github.com/andrewprock/pokerstove"
+  url "https://github.com/andrewprock/pokerstove/archive/v1.0.tar.gz"
+  sha256 "68503e7fc5a5b2bac451c0591309eacecba738d787874d5421c81f59fde2bc74"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    mkdir "build" do
+      system "cmake", "-DCMAKE_BUILD_TYPE=Release", ".."
```

Comment:
Use `std_cmake_args` instead of `CMAKE_BUILD_TYPE`. See other formulas for how it works.